src/Microsoft.Diagnostics.EventFlow.Inputs.Log4net/Log4netFactory.cs

```diff
@@ -0,0 +1,21 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using Microsoft.Extensions.Configuration;
+using Validation;
+
+namespace Microsoft.Diagnostics.EventFlow.Inputs
+{
+    public class Log4NetFactory : IPipelineItemFactory<Log4NetInput>
```

Comment:
You probably want to add information about `Log4NetFactory` to [`DiagnosticPipelineFactory`](https://github.com/Azure/diagnostics-eventflow/blob/master/src/Microsoft.Diagnostics.EventFlow.Core/Implementations/DiagnosticPipelineFactory.cs#L325)

After doing that please also increment the patch number for the EventFlow.Core assembly (`VersionPrefix` element in the project file).